examples/one_dot_one/bin/generate.mjs

```diff
@@ -1,3 +1,5 @@
+#!/usr/bin/env node
```

Comment:
I kept trying to use this as a shell script and it wasn't working. This is fine with the license down a few lines, right?

Reply:
Probably fine but NAL